package.json

```diff
@@ -64,6 +64,9 @@
       }
     }
   },
+  "mocha": {
+    "exit": true
```

Comment:
I think this is not required now

Reply:
Indeed

Reply:
Changed it in https://github.com/inmanta/vscode-inmanta/pull/33/commits/939de709d3b0da790beb077a77b621e46ae04848